tests/phpunit/tests/Admin/ImgAreaSelect_Patch.php

```diff
@@ -28,7 +28,13 @@
  *
  * @coversDefaultClass \Google\Web_Stories\Admin\ImgAreaSelect_Patch
  */
-class ImgAreaSelect_Patch extends Test_Case {
+class ImgAreaSelect_Patch extends TestCase {
+	public function tearDown() {
+		unset( $GLOBALS['current_screen'] );
```

Comment:
`WP_UnitTestCase` should be doing this already 🤔

Reply:
Yes, see https://core.trac.wordpress.org/ticket/53431